app.rb

```diff
@@ -71,14 +73,6 @@ def t(*args)
 Dir[File.dirname(__FILE__) + '/models/*.rb'].each { |file| require file }
 Dir[File.dirname(__FILE__) + '/presenters/*.rb'].each { |file| require file }
 
-
-$check_index_mapping_exists = defined?(RAKE_SEARCH) === nil || RAKE_SEARCH === false
-if $check_index_mapping_exists
-  # Ensure Elasticsearch index mappings exist, unless we are creating it in the rake search initialize
-  Comment.put_search_index_mapping
```

Comment:
For now, I deleted these `put_search_index_mapping` which were not needed because the mappings must be created during initialization.

This code was doubling as a way to fail fast if the "content" alias doesn't exist.  Since we now plan on having the separate initialization rake task, it is probably important to restore a check, which will require `RAKE_SEARCH` (or `IN_RAKE_SEARCH_INITIALIZE`) to skip the check during initialization.  I can make the check a simpler check for the 'content' alias, where we assume the mappings will exist.